wcbench.sh

```diff
@@ -717,10 +746,23 @@ issue_odl_config()
     # This could be done with public key crypto, but sshpass is easier
     if ! command -v sshpass &> /dev/null; then
         echo "Installing sshpass. It's used for issuing ODL config."
-        if "$VERBOSE" = true; then
-            sudo yum install -y sshpass
+
+        if [ "$OS" == 'fedora' ]; then     #Please check if "echo $OSTYPE" returns "fedora". If not replace "fedora" here with the returned name
+            if "$VERBOSE" = true; then
+                sudo yum install -y sshpass
+            else
+                sudo yum install -y sshpass &> /dev/null
+            fi
+       
+        elif [ "$OS" == 'linux-gnu' ]; then
```

Comment:
This is what Fedora OSs return, need a better check or to manually set the flag.